schemas/yazi.json

```diff
@@ -359,6 +337,31 @@
 					}
 				]
 			}
+		},
+		"rules": {
```

Comment:
Maybe use `open_rules` / `openRules` or just `open` or something? cc @uncenter do you have any preferences for it?

Reply:
`open_rules` is the pattern we use in uh the `theme.json` schema, let's use that here as well.